config/locales/en.yml

```diff
@@ -2,25 +2,32 @@
 en:
   global_popup_title: "Redmine Advanced Messenger"
   
-  issue_notifications_dialog_go_to_first_one: "Go to first one"
-  issue_notifications_dialog_text_for_current_user: "For %{user}, there are <span id='unread-notifications-count'></span> <span style='color: red'> unread </span> notifications."
-  issue_notifications_dialog_others_tooltip_text: "Unread notifications"
-  issue_notifications_dialog_others_tooltip_text_all_read: "Other users have all messages read."
-  issue_notifications_dialog_status_for_others: "Status for other users"
-  
-  journal_collapse_note_message: "The message is collapsed. Preview: "
-  journal_collapse_note_message_read: "The message is read, but collapsed. Preview: "
-  journal_collapse_note_message_unread: "The message is unread. Please expand it to see its content & mark read."
-  journal_notifications_tooltip_list_title: "%{date} notifications sent (<span style='color:green'>read</span>/<span style='color:red'>unread</span>) to:"
-  journal_notifications_tooltip_list_item_not_viewed: "unread"
-  journal_read: "Read"
-
   label_time_ago: "%{time} ago"
+  
+  message_collapsed: "The message is collapsed. Preview: "
+  message_collapsed_read: "The message is read, but collapsed. Preview: "
+  message_collapsed_unread: "The message is unread. Please expand it to see its content & mark read."
+  message_notifications_tooltip_list_title: "%{date} notifications sent (<span style='color:green'>read</span>/<span style='color:red'>unread</span>) to:"
+  message_notifications_tooltip_list_item_not_viewed: "unread"
+  message_read: "Read"
+  
+  notifications_dialog_go_to_first_one: "Go to first one"
+  notifications_dialog_text_for_current_user: "For %{user}, there are <span id='unread-notifications-count'></span> <span style='color: red'> unread </span> notifications."
+  notifications_dialog_others_tooltip_text: "Unread notifications"
+  notifications_dialog_others_tooltip_text_all_read: "Other users have all messages read."
+  notifications_dialog_status_for_others: "Status for other users"
 
-  my_page_unread_notifications: "Unread notifications"
-  my_page_all_notifications_read: "All notifications are read"
+  my_page_issues_unread_notifications: "Unread issues notifications:"
+  my_page_forum_unread_notifications: "Unread forum messages:"
+  my_page_all_issues_notifications_read: "All issues notifications are read"
+  my_page_all_forum_notifications_read: "All forum messages are read"
   
   top_app_menu_unread_notifications: "Unread notifications"
 
+  # Plugin settings messages
+  show_unread_notifications: Show "Unread notifications" in header bar
+  unread_notifications_update_interval: Update "Unread notifications" interval
+  unread_notifications_update_interval_note: seconds
```

Comment:
I saw that in the specification, but it was a little bit unclear. Because:
1. This particular message was not quoted, like the other were
![image](https://github.com/famiprog/redmine_advanced_messenger/assets/4996054/516996ef-f566-4dd2-a25e-21e4c0641934)

2. The field already has a default value of 60. I thought that maybe is a duplication to add also "E.g. 60."

So, what do you think, the text should be "E.g. 60. In seconds" or maybe just "In seconds"?

Reply:
I thing the more complex form "E.g. 60. In seconds" because it tells you how you should complete the field, not only the measurement unit.

If you find a more clear way to formulate, try. But it should contain what to write (60) and what it represents (seconds) and the fact that what you just write (60) is an example

Reply:
ok.